src/presentational/Dropdown.jsx

```diff
@@ -17,9 +21,10 @@ export default class Dropdown extends Component {
   render() {
     const options = this.props.options.map(option =>
         <option
-          key={option.value}
-          value={option.label}>
-            {option.label}
+          // the disjunctions allow for the case where options are classrooms (right-hand side)
```

Comment:
This probably isn't a great idea, as the end point is writing some mad conditional to accommodate everything that gets passed into it. Why not have a consistent API and modify the props passed in for classrooms to have a value/label structure instead?